articles/hosted-pages/login/index.md

```diff
@@ -0,0 +1,127 @@
+- [Lock Passwordless](/hosted-pages/login/lock-passwordless) - Lock Passwordless uses the same style of interface as Lock, but rather than offering identity providers as login options, will simply ask the user to enter an email or SMS number to begin a Passwordless authentication transaction.
+- [Auth0.js v8](/hosted-pages/login/auth0js/v8) - Auth0.js is the SDK used for interacting with the Auth0 authentication API. Primarily, you would use the SDK if you need to build your own custom login UI, or implement more complex functionality than simply allowing your users to login. Auth0.js v8 is the most up to date version of that SDK, and is fully capable of OIDC compliant transactions.
+- [Auth0.js v7](/hosted-pages/login/auth0js/v7) - Auth0.js is the SDK used for interacting with the Auth0 authentication API. Primarily, you would use the SDK if you need to build your own custom login UI, or implement more complex functionality than simply allowing your users to login. Version 7 is the older supported version, and is only for use by applications which are not yet ready to upgrade to version 8, or cannot be OIDC compatible.
+
+### 3. Customization
+
+You can customize the Hosted Login Page at will right from the editor. If you use Lock, you can alter its behavior and appearance with [customization options](/libraries/lock/v10/customization). If you are building a custom UI, you can style the Hosted Login Page to your own specifications.
+
+All changes to the page's appearance and/or behavior will apply to *all* users shown this login page, regardless of the client or connection. Remember that the Hosted Login Page customizations are per _tenant_ rather than per client.
+
+### Query String Parameters
+
+You can also pass [OIDC specification](http://openid.net/specs/openid-connect-core-1_0.html#AuthRequest) compliant parameters into the Hosted Login Page from your application by adding them to the query you use to call the `authorize` endpoint. This allows you to make further customizations to your Hosted Login page from the application. 
+
+If you intend to pass any configuration parameters to the page from your client, you may need to become familiar with the `config` object. The `config` object contains the set of configuration values that adjusts the hosted login page's behavior at runtime. Some of the values in `config` are parameters which are received from your application code to your Hosted Login Page.
+
+You can examine the contents of the `config` object using the following within the hosted login page editor:
+
+```js
+// Decode configuration options
+var config = JSON.parse(decodeURIComponent(escape(window.atob('@@config@@'))));
+```
+
+The below examples assume that you are using [Auth0.js](/libraries/auth0js/v8) to call the `authorize` endpoint from your application in order to show the Hosted Login Page.
```

Comment:
That doc is for using Auth0.js in the HLP; these examples are you using Auth0.js in your application from which you're calling the HLP. I basically just wanted to show people that you can pass those parameters to the page. 

If you object to using Auth0.js here, I think I'd rather build the URL manually than move the examples elsewhere, as it's pertinent to getting config information from your application _to_ the hosted page no matter what technology you use _inside_ the page.